Lib/test/test_syntax.py

```diff
@@ -199,6 +199,10 @@
 Traceback (most recent call last):
 SyntaxError: invalid syntax
 
+>>> for a, b
```

Comment:
If we backport this PR, are these compatible with the old parser?

Reply:
Yup yup, they should be.